### PR TITLE
feat: add mouseX and mouseY

### PR DESCRIPTION
**Pull request by hawkingrei**
😈   I add new keywords `mouseX` and `mouseY`. You can test by the following R code
```R
draw <- function() {
    background(as.integer(204))
    line(mouseX, mouseY, 80, mouseY);
}
```

### CODE REVIEW

**Comment by gaocegege** [comment]
Hi, there is one conflicting file, please resolve it 😄

File: src/rprocessing/RLangPApplet.java [hunk 1]
```diff
@@ -342,18 +342,31 @@ public void focusLost() {
...
    */
   protected void wrapProcessingVariables() {
     log("Wrap Processing built-in variables into R top context.");
     // TODO: Find some ways to push constants into R.
+    wrapMouseVariables();
     this.renjinEngine.put("width", width);
     this.renjinEngine.put("height", height);
     this.renjinEngine.put("displayWidth", displayWidth);
     this.renjinEngine.put("displayHeight", displayHeight);
-    // this.renjinEngine.put("focused", focused);
```
Comment (gaocegege) [inline src/rprocessing/RLangPApplet.java]: I think the conflict is caused by this statement

Comment (jeremydouglass) [inline src/rprocessing/RLangPApplet.java]: Yes, I enabled that line in 912d23b -- before, `processing$focused` worked, but `focused` did not. Now `focused` works. We should be able to merge the mouseX / mouseY code and leave the line -- unless I'm misunderstanding something.

Comment (hawkingrei) [inline src/rprocessing/RLangPApplet.java]: resolved

Comment (gaocegege) [inline src/rprocessing/RLangPApplet.java]: @jeremydouglass I think you are right 😄 

And LGTM

**Comment by gaocegege** [comment]
LGTM

**Pull request merged**